src/app/pages/link1/link1.component.html

```diff
@@ -1 +1 @@
-<p>Link 1 page content goes here..</p>
+<p>link1 works!</p>
```

Comment:
Why not stay consistent with the content on other pages?

Reply:
I will correct.